### PR TITLE
Fix ESLint warnings and errors

### PR DESCRIPTION
**Pull request by Saibamen**
# Description

Fix ESLint warnings and errors

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Other

## Checklist

- [ ] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [ ] I have performed a self-review of my own code and tested it
- [ ] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [ ] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

image

### CODE REVIEW

File: src/pages/StatusPage.vue [hunk 1]
```diff
@@ -273,30 +274,29 @@
             {{ $t("deleteStatusPageMsg") }}
         </Confirm>
 
-        <component is="style" v-if="config.customCSS" type="text/css">
+        <component :is="style" v-if="config.customCSS" type="text/css">
```
Comment (louislam) [inline src/pages/StatusPage.vue:277]: I think the original one is correct, `style` is a string.

After adding a `:`, it probably becomes a undefined variable `style`.

Recommend to ignore the eslint rule if they complain this.

Comment (Saibamen) [reply src/pages/StatusPage.vue:277]:
```
[vue/require-component-is]
Expected '<component>' elements to have 'v-bind:is' attribute
```

And we have 2 other places when we are using `:is`:
image

Comment (louislam) [reply src/pages/StatusPage.vue:277]: Just tested, this change actually bugged as I thought.

image

Comment (Saibamen) [reply src/pages/StatusPage.vue]: Reverted. Thanks for testing

**Review by gaby — APPROVED** [review]

**Pull request merged**